packages/odyssey-react-mui/src/themes/odyssey/palette.ts

```diff
@@ -20,7 +20,7 @@ export const palette: ThemeOptions["palette"] = {
   },
   primary: {
     light: "#a7b5ec",
-    main: "#ff4400",
+    main: "#1662dd",
```

Comment:
Unlike the other section, this looks like a pretty major color change. It's white to some darker color. Blue? What's the change look like?

Reply:
This is the proper color; it hadn't been updated previously.